src/System Application/App/Email/src/Email/Inbox/EmailInboxViewer.Page.al

```diff
@@ -0,0 +1,197 @@
+                            Importance = Additional;
+                        }
+
+                        field(BccField; BccRecipient)
+                        {
+                            Caption = 'Bcc';
+                            ApplicationArea = All;
+                            ToolTip = 'Specifies the email BCC recipients.';
+                            Editable = false;
+                            Importance = Additional;
+                        }
+
+                        field(SubjectField; EmailSubject)
+                        {
+                            Caption = 'Subject';
+                            ApplicationArea = All;
+                            ToolTip = 'Specifies the subject of the sent email.';
+                            Editable = false;
+                            Importance = Promoted;
+                        }
+                    }
+                }
+            }
+
+            group(HTMLFormattedBody)
+            {
+                ShowCaption = false;
+                Caption = ' ';
+                Visible = IsHTMLFormatted;
+
+                field("Email Editor"; EmailBody)
+                {
+                    Caption = 'Message';
+                    ApplicationArea = All;
+                    ToolTip = 'Specifies the content of the email.';
+                    MultiLine = true;
+                    ExtendedDatatype = RichContent;
+                    Editable = false;
+                }
+            }
+
+            group(RawTextBody)
+            {
+                ShowCaption = false;
+                Caption = ' ';
+                Visible = not IsHTMLFormatted;
+                field(BodyField; EmailBody)
+                {
+                    ApplicationArea = All;
+                    ShowCaption = false;
+                    Caption = 'Message';
+                    ToolTip = 'Specifies the content of the email.';
+                    MultiLine = true;
+                    Editable = false;
+                }
+            }
+
+
+            part(Attachments; "Email Attachments")
+            {
+                ApplicationArea = All;
+                SubPageLink = "Email Message Id" = field("Message Id");
+                UpdatePropagation = SubPart;
+                Caption = 'Attachments';
+                Visible = HasAttachments;
+            }
+        }
+    }
+
+    actions
+    {
+        area(Processing)
+        {
+            action(ShowSourceRecord)
+            {
+                ApplicationArea = All;
+                Image = GetSourceDoc;
+                Caption = 'Show Source';
+                ToolTip = 'Open the page from where the email was sent.';
+                Promoted = true;
+                PromotedCategory = Process;
+                PromotedOnly = true;
+                Enabled = HasSourceRecord;
+
+                trigger OnAction()
+                begin
+                    EmailImpl.ShowSourceRecord(Rec."Message Id");
```

Comment:
Does this make sense for Email Inbox? Wouldn't this always error

Reply:
Referring to the action. Do we ever add source records to outbox messages?

Reply:
No idea tbh, I never tried any of the actions. Purely used this page for testing, as I showed you the broken looking design when I was using it. With no time to get Jacob and team's design review atm, I'm opting to just remove these for the first iteration.